integration-test-example.js

```diff
@@ -0,0 +1,24 @@
+// Integration test file with intentional issues for commenting
+function processUserData(users) {
+    // Missing input validation
+    let results = [];
+
+    // SQL injection vulnerability
+    const query = "SELECT * FROM users WHERE id = " + users[0].id;
+
+    // Hardcoded secrets
+    const apiKey = "sk-1234567890abcdef";
```

Comment:
🔧 API key exposed in code! What do you think about using environment variables instead? This prevents secrets from being committed to git.